# DO NOT MODIFY THIS FILE, IT IS AUTO GENERATED
agorum.workflow.translation.auto.agorum.dev.assert.fail.1.fail.displayName=Fail
agorum.workflow.translation.auto.agorum.dev.assert.fail.1.workflow.description=Einen Fehler werfen, um den Workflow-Prozess zu unterbrechen und einen Fehler auszugeben.
agorum.workflow.translation.auto.agorum.dev.assert.fail.1.workflow.documentation=<h1>Knoten &quot;Fehler&quot; (agorum.dev.assert.fail)</h1>

<p>Dieser Knoten erzeugt einen Fehler und bricht den aktuellen Workflow-Lauf ab.</p>

agorum.workflow.translation.auto.agorum.dev.assert.fail.1.workflow.displayName=Fehler
agorum.workflow.translation.auto.agorum.dev.assert.fail.1.configuration-widget.configuration.message.title=Fehlermeldung
agorum.workflow.translation.auto.agorum.dev.assert.fail.1.configuration-widget.configuration.message.help=Fehlermeldung, die ausgegeben werden soll.
agorum.workflow.translation.auto.agorum.dev.assert.fail.1.configuration-widget.configuration.message.placeholder=Text
agorum.workflow.translation.auto.agorum.dev.assert.fail.1.fail.enter.message.default=Keine Fehlermeldung definiert
